06-aks-cluster.md

```diff
@@ -9,19 +9,19 @@ Now that your [ACR instance is deployed and ready to support cluster bootstrappi
    > If you cloned this repo, then the value will be the original mspnp GitHub organization's repo, which will mean that your cluster will be bootstraped using public container images. If instead you forked this repo, then the GitOps repo will be your own repo, and your cluster will be bootstrapped using container images references based on the values in your repo's manifest files. On the prior instruction page you had the opportunity to update those manifests to use your ACR instance. For guidance on using a private bootstrapping repo, see [Private bootstrapping repository](./cluster-manifests/README.md#private-bootstrapping-repository).
 
    ```bash
-   GITOPS_REPOURL=$(git config --get remote.origin.url)
-   echo GITOPS_REPOURL: $GITOPS_REPOURL
+   export GITOPS_REPOURL_AKS_BASELINE=$(git config --get remote.origin.url)
```

Comment:
Why are we exporting now?

Reply:
because https://github.com/mspnp/aks-baseline/blob/main/saveenv.sh save the variables that are exported and end by _AKS_BASELINE. 
Without the export, the variables are not saved. 
That is good when you for some reason need to re-start a deploy process in the middle, you have the variable values till the moment you executed the latest  saveenv.sh.